iOS Client/Utilities/Formatters.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Formatters.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 11/14/17.
+//  Copyright © 2017 Mockingbird. All rights reserved.
+//
+
+import Foundation
+
+struct Formatters {
+    
+    static let devotionDateFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "yyyy-MM-DD"
+        return formatter
+    }()
+    
+    static let calendar: NSCalendar? = {
+       return NSCalendar.init(calendarIdentifier: NSCalendar.Identifier.gregorian)
+    }()
+}
```

Comment:
If we compute the DateComponents as a lazy property in the MBDevotions class, then we could move these details into that class as well, either as private helper methods or private lazy properties, since they are specific to the MBDevotion objects.